.github/workflows/ci.yml

```diff
@@ -0,0 +1,44 @@
+name: CI
+on: push
+jobs:
+  test:
+    name: Run tests
+    runs-on: macos-latest
```

Comment:
Do we want to use a specific mac-os instead of macos-latest ? (For some reason macos-latest is 10.15 ? )

Reply:
looks like this is why macos-latest is 10.15, but i'll switch to explicitly run on 10.15 so we don't get any surprises when the default switches to 11.0 
https://github.com/actions/virtual-environments
> The macOS 11.0 virtual environment is currently provided as a private preview only.
> The "macos-latest" YAML workflow label still uses the macOS 10.15 virtual environment.